hphp/hack/src/stubs/dune

```diff
@@ -65,16 +65,11 @@
  (modules saved_state_loader state_loader_futures state_loader_lwt)
  (libraries
   hg
+  server_env
   typechecker_options)
  (preprocess
   (pps lwt_ppx ppx_deriving.std)))
 
- ; this stub file is used by both buck2 and dune
-(rule
- (target remote_old_decls_ffi.ml)
- (action
-  (copy# ../facebook/remote_old_decls/stubs/remote_old_decls_ffi.ml remote_old_decls_ffi.ml)))
```

Comment:
Alternatively if upgrading to Dune 3.17.0+ is fine, there's a `file-available` macro.